src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -320,9 +327,19 @@ export function CollapsibleNavGroupEnabled({
             hasShadow={false}
             className="eui-yScroll flex-1-container"
           >
-            {shouldShowCollapsedNavHeaderContent && collapsibleNavHeaderRender ? (
+            {shouldShowCollapsedNavHeaderContent &&
+            collapsibleNavHeaderRender &&
+            !currentWorkspace ? (
               <>
-                {collapsibleNavHeaderRender()}
+                <EuiPanel
+                  paddingSize="none"
+                  color="transparent"
+                  hasBorder={false}
+                  hasShadow={false}
+                  style={{ height: '40vh' }}
+                >
+                  {collapsibleNavHeaderRender()}
```

Comment:
@SuZhou-Joe can we remove workspace list here now if left nav on homepage will anyway not be displayed soon?

Reply:
Agree, let's remove the list content here. Only show workspace picker when inside a use case.